src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,6 +9,7 @@
  */
 public class Tag {
 
+    public static final String INVALID_TAG = "Please enter valid tag(s).";
```

Comment:
May be better to rename this field (along with its usages) to `MESSAGE_INVALID_TAG`, else it may be confused with the static constants defined in `seedu.address.logic.commands.CommandTestUtil`.